src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -2571,7 +2577,7 @@ protected function generateTree($path, $intMargin, $mount=false, $blnProtected=t
 				{
 					--$countFiles;
 				}
-				elseif ($this->blnHideFiles && !is_dir(TL_ROOT . '/' . $currentFolder . '/' . $file))
+				elseif (!$this->blnShowFiles && !is_dir(TL_ROOT . '/' . $currentFolder . '/' . $file))
```

Comment:
That's how it was always implemented. It's `eval => ['files' => true]`, and by default it's `false`.